### PR TITLE
Bloyal 221209

### PR DESCRIPTION
**Pull request by brianloyal**
*Description of changes:*
Copy editing and review of HIV and tabular breast cancer workshops.

Made the following updates:
## HIV workshop
- Reduced the number of training instances to 1 to stay below default quota.
- Reduced the number of training epochs to 20 to finish faster (validation results are still pretty good)
- Swapped out the `cat!` command to a `%pycat` to get syntax formatting of training script
- Copy editing

## Brest Cancer workshop
- Added the new sagemaker_datawrangler library to make pandas viewing pretty
- Copy editing

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by shamika — CHANGES_REQUESTED** [review]
Added few comments

File: workshops/Molecular-property-prediction/hiv-inhibitor-prediction-dgl/molecule-hiv-inhibitor-prediction-sagemaker.ipynb [hunk 1]
```diff
@@ -1289,7 +1087,7 @@
     "    role = role,\n",
     "    framework_version = \"1.9.0\",\n",
     "    py_version=\"py38\",\n",
-    "    instance_count=2,\n",
+    "    instance_count=1,\n",
     "    instance_type=\"ml.p3.2xlarge\",\n",
```
[the remark this answers is not in the document]
Comment (brianloyal) [reply workshops/Molecular-property-prediction/hiv-inhibitor-prediction-dgl/molecule-hiv-inhibitor-prediction-sagemaker.ipynb:1091]: Yeah, it seems to work fine on a single p3.2x. I was surprised that g4dns aren't available for training jobs by default

File: workshops/Bring_Your_Own_Sklearn_Classifier/byo_mlp_classifier/sklearn_bring_your_own_MLP_Classifier_Breast_Diagnostic.ipynb [hunk 2]
```diff
@@ -1,42 +1,59 @@
...
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Ensure we have the right version of sagemaker\n",
-    "%pip install sagemaker==2.48.0"
+    "%pip install -U sagemaker>=2.48.0"
```
[the remark this answers is not in the document]
Comment (brianloyal) [reply workshops/Bring_Your_Own_Sklearn_Classifier/byo_mlp_classifier/sklearn_bring_your_own_MLP_Classifier_Breast_Diagnostic.ipynb:48]: I'm cool with that - I'll update mine as well

**Comment by brianloyal** [comment]
Shamika reviewed and approved

**Pull request merged**